Riot.xcodeproj/project.pbxproj

```diff
@@ -6912,7 +6907,9 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_GROUP_IDENTIFIER = group.au.gov.act.aether;
```

Comment:
Can all references to au.gov.act.aether please be replaced with au.gov.act.health.aether

Reply:
All references in the project have been updated to au.gov.act.health.aether (with appropriate prepend / append values).